sample/AspNetFullFrameworkSampleApp/MyModule1.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Elastic.Apm.DiagnosticSource;
+using System;
+using System.Web;
+using Elastic.Apm;
+
+namespace AspNetFullFrameworkSampleApp
+{
+	public class MyModule1 : IHttpModule
```

Comment:
Could we maybe rename this to something more meaningful before we merge to master? (like ElasticApmModul, or something similar)